core/src/main/scala/com.snowplowanalytics.snowplow.collector.core/Run.scala

```diff
@@ -98,7 +98,6 @@ object Run {
           config.enableDefaultRedirect,
           config.rootResponse.enabled,
           config.crossDomain.enabled,
-          config.networking.responseHeaderTimeout,
```

Comment:
Now I noticed - we never passed `bodyReadTimeout`? Is it intentional?

Reply:
Yea. In the end there was no difference between the two. Removing makes much more sense.